test/client/fixtures/sandbox/code-instrumentation/process-script-test.js

```diff
@@ -78,3 +78,14 @@ test('should not process non-string args', function () {
     deepEqual(execScript('eval.apply(this, [[1, 2 ,3]])'), [1, 2, 3]);
 });
 
+module('var foo = eval');
+
+test('The eval method should switch its own context to global', function () {
+    throws(function () {
```

Comment:
We need more test cases:
1)return eval from the scope - it's shouldn't have closure
2)pass eval as the func argument - it's shouldn't have closure on the upper scope